configuration_version_test.go

```diff
@@ -83,11 +83,13 @@ func TestConfigurationVersionsCreate(t *testing.T) {
 			wTest.ID,
 			ConfigurationVersionCreateOptions{},
 		)
+		assert.NotEmpty(t, cv.UploadURL)
 		require.NoError(t, err)
 
 		// Get a refreshed view of the configuration version.
 		refreshed, err := client.ConfigurationVersions.Read(ctx, cv.ID)
 		require.NoError(t, err)
+		assert.Empty(t, refreshed.UploadURL)
 
```

Comment:
For this test, the fist time you create a ConfigVersion, you should have access to the upload-url, but the second time, it should be empty.